at_command.c

```diff
@@ -733,8 +735,11 @@ EXPORT_DEF int at_enqueue_retrieve_sms(struct cpvt *cpvt, int index)
 		return -1;
 	}
 
+	if (suppress_error == SUPPRESS_ERROR)
+		cmds[0].flags |= ATQ_CMD_FLAG_SUPPRESS_ERROR;
```

Comment:
Please always use curly brackets around conditionals.

See the Asterisk style guide.